packages/studiocms_core/src/i18n/LanguageSelector.astro

```diff
@@ -0,0 +1,103 @@
+---
+import { languageSelectorOptions, switchLanguage } from './index';
+
+const switcher = switchLanguage(Astro);
+
+// TODO: Update the language selector's styles to match StudioCMS's new design
+---
+
+<button
+    id="language-selector"
+    class="language-selector mr-2"
+    >
+    <svg xmlns="http://www.w3.org/2000/svg" class="w-full h-[80%]" width="1.5rem" height="1.5rem" viewBox="0 0 24 24"><g fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"><path d="M4 5h7M9 3v2c0 4.418-2.239 8-5 8"/><path d="M5 9c0 2.144 2.952 3.908 6.7 4m.3 7l4-9l4 9m-.9-2h-6.2"/></g></svg>
+</button>
+
+<menu id="language-sector-menu" type="context" class="bg-neutral rounded-md p-2">
+    {
+        //Language Buttons Auto-Generated
+        languageSelectorOptions.map(({ key, value }) => (
+            <a class="btn btn-primary" href={switcher(key)}>{value}</a>
+        ))
+    }
+    <div class="flex flex-row-reverse">
+        <button 
+            id="language-sector-menu-cancel" 
+            class="btn btn-sm btn-circle btn-outline text-neutral-content" 
+            type="button">
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-4 w-4" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+            </svg>
```

Comment:
Again, inlined SVG

Reply:
There is a comment about this component and i've said before, its not yet implemented, and needs to be updated to match the StudioCMS layout.  this is copied from one of my other projects.